src/widgets/heightandweight/heightandweight-record.component.tsx

```diff
@@ -76,7 +76,7 @@ export default function HeightAndWeightRecord(
                   <tr>
                     <td>Weight</td>
                     <td>
-                      {dimensions.weight} <span>Kg</span>
+                      {dimensions.weight} <span>kg</span>
```

Comment:
Can we store these constant labels somewhere (e.g., in a constants.ts file)?

I assume that the API always gives us `kg`, right?

Reply:
Above we use `dimensions?.obsData` here `dimensions.weight`. Can `dimensions` be `undefined` or not?

Reply:
It can be. I've rectified this.

Reply:
> Can we store these constant labels somewhere (e.g., in a constants.ts file)?
> 
> I assume that the API always gives us `kg`, right?

I think so. I'm not sure where the appropriate location for this would be. Perhaps @nkimaina can chime in on this?

Reply:
Ping @nkimaina - thanks!